scripts/zap-scanner.sh

```diff
@@ -67,13 +66,15 @@ else
     ZAP_ARGS+=(-c "$CONFIG_FILE")
 fi
 
-# TODO: Remove this after all open alerts are categorized in the config files
-# Don't trigger a failure on WARNING level alerts, until we have a complete config file this will cause us to never
-# fail this step. However, if we do fail on warnings with the current configuration state every build will fail.
-# This is because ZAP defaults all alerts to WARN level, and any warnings found will trigger a failing exit code
-# from the zap-full-scan.py script.
+# Prevent failures for alerts triggered at WARN level.
+# At this time rules at this level are known issues to correct, but if ZAP
+# adds new rules we don't have categorized they will default to WARN.
 ZAP_ARGS+=(-I)
 
+# Use custom hook to disable passive scan rules - these don't get disabled by
+# setting them to IGNORE in the config file, unlike active rules.
+ZAP_ARGS+=(--hook=/zap/wrk/zap-hook.py)
```

Comment:
This is the important change for this file, we must pass in the hook script to ignore passive rules.